src/pages/courseDetailsPage/CourseDetailsPage.tsx

```diff
@@ -71,7 +71,7 @@ const CourseDetailsPage: React.FC = () => {
             videoLink={currentLesson.link}
             order={currentLesson.order}
             previewImageLink={currentLesson.previewImageLink}
-            status={currentLesson.status}
+            status={currentLesson.status === "locked" ? "locked" : "unlocked"}
```

Comment:
You already have constant for "locked" status. Maybe shall write `currentLesson.status === LOCKED` ?